ppr-ui/src/components/mhrRegistration/SubmittingParty/MhrSubmittingParty.vue

```diff
@@ -29,6 +29,13 @@
 
         <v-divider class="my-9 ml-0 mr-2" />
 
+        <v-card v-if="showImportantMsg" outlined id="important-message" class="rounded-0 mb-9">
```

Comment:
Can you check directly for `v-if="businessName"` here instead?

Reply:
not really, this message is only shown when we retrieve data from `partySearch`.
This message isn't shown when just switching from individual person to business.

Reply:
Just to elaborate here, this messages requirements are 2-fold
- is a business
- and have they applied the values using the lookup

So yea, there is a little bit extra to the condition. 

That being said, could we update the property name to explain the property a bit more. 
ie `isBusinessLookup` or something along those lines.

Naming is hard :)

Reply:
changed the name to `isBusinessLookup`